### ISSUE TITLE
Handle case when keyring password does not work

### ISSUE THREAD

**Issue by heroldus** [ImmobilienScout24/afp-cli#40]
The keyring feature in #27 is really great. The only pitfall is that the password expiration will come soon. To tackle that you could re-prompt for a password if the authentication on the afp-server fails and save the new password in keyring

**Comment by esc** [comment]
Yes, we should think about improving the handling logic. Right now, it works 80% of the time but it would be nice to make it smarter and less, shall I say "hacky".

**Comment by schlomo** [comment]
A simpler solution might be to simple remove the cached password from the keyring if it did not work, together with a suitable message to the user like "Your cached password did not work - removing it. Please try again".

That would follow a "clean up after yourself" mind set without the need for a password changed dialog.

**Comment by schlomo** [comment]
ATM the workaround is to manually go to the password keyring and change/delete the password there. IMHO this is too non-obvious.

**Comment by schlomo** [comment]
Idea: Simply remove the stored password from keyring if it does not work.

This also covers other cases like the user entering a wrong password or unforeseen troubles.
